calico/networking/ip-autodetection.md

```diff
@@ -140,7 +203,40 @@ In the following scenarios, you may want to configure a specific IP and subnet:
 - Changes to cross subnet packet encapsulation
 - Changes to host IP address
 
-You can configure specific IP address and subnet for a node using environment variables or by updating the [Node resource]({{ site.baseurl }}/reference/resources/node).
+{% tabs %}
+  <label:Operator,active:true>
+<%
+##### Configure IP and subnet using default Installation resource
+
+To configure the IP addresses used by the node, edit the default [Installation]({{site.baseurl}}/reference/installation/api#operator.tigera.io/v1.Installation)
```

Comment:
This section isn't yet supported by the operator (and may not be, if there's no need for it).

I suggest adding a note to that extent within this tab:

> Manually specifying IP addresses for individual nodes is not supported when installing via the operator. Please use one of the supported IP autodetection methods above instead.